src/assistant_library_with_local_commands_demo.py

```diff
@@ -61,14 +66,17 @@ def process_event(assistant, event):
         status_ui.status('listening')
 
     elif event.type == EventType.ON_RECOGNIZING_SPEECH_FINISHED and event.args:
-        text = event.args['text']
-        print('You said:', text)
-        if text == 'power off':
+        print('You said:', event.args['text'])
+        text = event.args['text'].lower()
+        if text == 'raspberry power off':
             assistant.stop_conversation()
             power_off_pi()
-        elif text == 'reboot':
+        elif text == 'raspberry reboot':
```

Comment:
Please revert the addition of "raspberry". Since this is just a sample and everyone has different preferences, I prefer to keep it simple.